rusk/src/lib/encoding.rs

```diff
@@ -202,3 +205,24 @@ impl From<SpentTransaction> for rusk_proto::ExecutedTransaction {
         }
     }
 }
+
+impl From<(Note, Note)> for rusk_proto::ExecutedTransaction {
```

Comment:
I've a bit of mixed feeling about that, as we discussed, since it's not exactly true than any Note could be converted into an `ExecutedTransaction`. So I would suggest either to drop the conversion, and do it manually – at the end it's happen only in one single place – where we know for sure this conversion is legit, or implement a `TryFrom`.

Reply:
I understand. It doesn't feel perfectly right to me either. I'll move the code back. A `TryFrom` wouldn't really fit here anyway.